here-naksha-lib-core/src/test/java/com/here/naksha/lib/core/models/storage/StringCodec.javabak

```diff
@@ -44,8 +44,8 @@ class StringCodec extends FeatureCodec<String, StringCodec> {
 
   @Override
   public @NotNull StringCodec encodeFeature(boolean force) {
-    JbFeatureDecoder jbFeatureDecoder = new JbFeatureDecoder(new JbDictManager()).mapBytes(featureBytes, 0, featureBytes.length);
-    Map<String, Object> featureAsMap = (Map<String, Object>) new JbMapDecoder().mapReader(jbFeatureDecoder.getReader()).toIMap();
+    JbRecordDecoder jbRecordDecoder = new JbRecordDecoder(new JbDictManager()).mapBytes(featureBytes, 0, featureBytes.length);
+    Map<String, Object> featureAsMap = (Map<String, Object>) new JbMapDecoder().mapReader(jbRecordDecoder.getReader()).toIMap();
```

Comment:
Why would be need javabak file?
If you know we need it - leave it. If you know that we don't - delete. If you don't know - add a TODO comment plz :)

Reply:
I would prefer to keep all of these as reference. There might be some functionalities that have not been ported into our new Kotlin classes, hence in some other unmigrated modules these will be called and result in compiler errors. It's better to have these to check what were actually performed underneath.